pkg/podmanager/pod_lifecycle_manager.go

```diff
@@ -116,6 +119,7 @@ func (manager StandardPodLifecycleManager) CanDeletePods(_ context.Context, admi
 // UpdatePods updates a list of Pods to match the latest specs.
 func (manager StandardPodLifecycleManager) UpdatePods(ctx context.Context, r client.Client, _ *fdbv1beta2.FoundationDBCluster, pods []*corev1.Pod, _ bool) error {
 	for _, pod := range pods {
+		logr.FromContextOrDiscard(ctx).V(1).Info("Deleting pod", "name", pod.Name)
```

Comment:
I would prefer to pull the `logr.FromContextOrDiscard(ctx)` call out an assign it to a variable instead of doing the same work multiple times in a loop.